### ISSUE TITLE
Migration to 2.0

### ISSUE THREAD

**Issue by loleg** [flathub/ch.threema.threema-web-desktop#12]
Threema has a beta 2.0 client, which has an official distribution on Flatpak rather than RPM/DEB as in the past. We should at least put a link to it once it is published, if not already help them get it into the software catalog and deprecate this one.

The following instructions provided on the official [download page](https://threema.ch/en/download):

> We distribute the beta version of Threema 2.0 for desktop through [Flatpak](https://www.flatpak.org/). Make sure you [have set up Flatpak for your distribution](https://www.flatpak.org/setup/). Run the following commands in your terminal:

```
$ flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
$ flatpak install --from https://releases.threema.ch/flatpak/threema-desktop/ch.threema.threema-desktop.flatpakref
```

> Note: Update the Threema beta version for desktop through your software manager or by running flatpak update in your terminal. For security reasons, Threema Beta doesn’t have full access to your files, which is why “drag and drop” is restricted in some cases. To allow full access, please run the following command:

`$ flatpak override ch.threema.threema-desktop --filesystem=host`

**Comment by ljrk0** [comment]
Absolutely! Unfortunately the 2.0 client right now only works for iOS users, hence this package. I'm currently just too swamped to deal with updating this r/n and since it's not a security risk (the client warns that it's out-of-date) it's lower prio for me. I hope to fix it soon...

**Comment by threema-danilo** [comment]
Hi folks! Thanks for maintaining this community package. As you probably saw, we already distribute Threema Desktop through a custom Flatpak repository. Our mid-term goal is to distribute via FlatHub instead and officially maintain the package there. There are some few hurdles left though, most importantly the font licensing that we're still about to sort out in the coming few months (hopefully it won't be long).

Since this package here is called `ch.threema.threema-web-desktop`, we should have no issues with conflicts, since the 2.x version is called `ch.threema.threema-desktop`. As @ljrk0 already mentioned, Threema for desktop 2.x is currently only compatible with iOS devices (Android support will hopefully be released in the near future), so for now both versions (1.x and 2.x) will have to coexist.

Once Desktop 2.x is compatible with both mobile platforms, this package can be deprecated.

If you have any questions about packaging, feel free to ping me about it.

**Comment by m-terlinde** [comment]
Hey @threema-danilo,

> Once Desktop 2.x is compatible with both mobile platforms, this package can be deprecated.

Is this already the case so that your 2.0 could be officially pushed to FH?

**Comment by threema-danilo** [comment]
@m-terlinde no, not yet. Right now the Flatpak is available through our own repository (see https://three.ma/md). We do plan to publish it via FlatHub as well, but haven't prioritized it yet.

**Comment by erebion** [comment]
> Once Desktop 2.x is compatible with both mobile platforms, this package can be deprecated.

Not really, the new Flatpak package built and provided by Threema does not support the aarch64 architecture.

That means I cannot even use it on my machine.

I'm currently waiting for aarch64 builds. I really hope the build process will get amended for aarch64.
